### PR TITLE
Namespace keys and remove key password

### PR DESCRIPTION
**Pull request by warrenbailey**

### CODE REVIEW

File: app/decrypter.py [hunk 1]
```diff
@@ -25,14 +25,14 @@ class DecryptError(Exception):
 
 
 class Decrypter(object):
-    def __init__(self, public_key, private_key, private_key_password):
+    def __init__(self, public_key, private_key, ):
```
Comment (JamesGardiner) [inline app/decrypter.py]: Trailing comma

File: app/tests/test_settings.py [hunk 2]
```diff
@@ -3,8 +3,7 @@
 from app import settings
 
 # ras keys
-RAS_SEFT_PRIVATE_KEY = settings.get_key(os.getenv('TEST_RAS_SEFT_PUBLIC_KEY', "./test_keys/sdc-seft-signing-ras-private-key.pem"))
-RAS_SEFT_PRIVATE_KEY_PASSWORD = os.getenv("TEST_PRIVATE_KEY_PASSWORD", "digitaleq")
+RAS_SEFT_CONSUMER_PRIVATE_KEY = settings.get_key(os.getenv('TEST_RAS_SEFT_PUBLIC_KEY', "./test_keys/sdc-seft-signing-ras-private-key.pem"))
```
Comment (JamesGardiner) [inline app/tests/test_settings.py]: Does this line break flake8?

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 sdx-common==0.7.0
 structlog==17.2.0
 cryptography==1.9
-PyJWT==1.5.2
+PyJWT==1..2
```
Comment (JamesGardiner) [inline requirements.txt]: Wrong

File: app/tests/test_settings.py (hunk 2 above)
Comment (warrenbailey) [inline app/tests/test_settings.py]: No

**Pull request merged**